### PR TITLE
Order settlement keeper test

### PR DESCRIPTION
**Pull request by 0xjocke**
- Update OrderSettledEvent assertions
- Remove *1e9
- Move order fees to separate test suite
- Use calcTransactionCostInUsd
- Apply keeperFeeBufferUsd correctly in test calculations
- Instead of manipulating eth price, manipulate configs to test bounded values

### CODE REVIEW

File: markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        const { PerpMarketProxy } = systems();
+
+        // Cap the max keeperFee to $50 USD
+        const maxKeeperFeeUsd = bn(50);
+        await setMarketConfiguration(bs, { maxKeeperFeeUsd, minKeeperFeeUsd: bn(10) });
+
+        const { trader, marketId, market, collateral, collateralDepositAmount } =
+          await depositMargin(bs, genTrader(bs));
+        const order = await genOrder(bs, market, collateral, collateralDepositAmount, {
+          desiredKeeperFeeBufferUsd: 1000, // Set a really high keeper buffer fee so we hit the max amount.
```
Comment (davidvuong) [inline markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts:258]: Would like to see a test where the keeperFee paid wasn't inflated by a large `desiredKeeperFeeBufferUsd` but rather due to expensive gas.

**Review by davidvuong — APPROVED** [review]
Left a comment but lgtm

File: markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+  setMarketConfigurationById,
+} from '../../helpers';
+
+describe('OrderModule fees', () => {
+  const bs = bootstrap(genBootstrap());
+  const { systems, restore, provider, collateralsWithoutSusd, markets, traders } = bs;
+
+  beforeEach(restore);
+
+  describe('getOrderFees', () => {
```
[the remark this answers is not in the document]
Comment (0xjocke) [reply markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts:25]: Correct only keeper tests

File: markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts (hunk 1 above)
Comment (0xjocke) [reply markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts:258]: Yea I was running into weird anvil edgecases with gas, this felt more stable. But maybe its more stable with the hardcoded gaslimits for settle and commit.

Comment (0xjocke) [reply markets/bfp-market/test/integration/modules/OrderModule.fees.test.ts:258]: Fixed

**Pull request merged**